worker/instancemutater/mocks/base_mock.go

```diff
@@ -41,15 +40,13 @@ func (m *MockAPICaller) EXPECT() *MockAPICallerMockRecorder {
 
 // APICall mocks base method
 func (m *MockAPICaller) APICall(arg0 string, arg1 int, arg2, arg3 string, arg4, arg5 interface{}) error {
-	m.ctrl.T.Helper()
```

Comment:
Are you using the latest _mockgen_? I seem to recall these being added when I updated it; now they are being removed...

Reply:
Hmm... apparently not! Let me pull down the latest and recreate these...

Reply:
You were right. It did put the `.Helper` bits back in although it recreated most mocks so the PR size ballooned a bit.